paper/paper.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'scikit-COSMO: A Toolbox of Machine Learning Methods for Materials Science'
+tags:
+    - Python
+authors:
+    - name: Rose K. Cersonsky ^[\url{mailto:rose.cersonsky@epfl.ch}]
+      orcid: 0000-0003-4515-3441
+      affiliation: 1
+    - name: Guillaume Fraux
+      orcid: 0000-0003-4824-6512
+      affiliation: 1
+    - name: Sergei Kliavinek
+      orcid: 0000-0001-8326-325X
+      affiliation: 1
+    - name: Alexander Goscinski
+      orcid: 0000-0001-8076-215X
+      affiliation: 1
+    - name: Benjamin A. Helfrecht
+      orcid: 0000-0002-2260-7183
+      affiliation: 1
+    - name: Michele Ceriotti ^[\url{mailto:michele.ceriotti@epfl.ch}]
+      orcid: 0000-0003-2571-2832
+      affiliation: 1
+affiliations:
+    - name: Laboratory of Computational Science and Modeling (COSMO), École Polytechnique Fédérale de Lausanne(EPFL), Lausanne, Switzerland
+      index: 1
+date: 21 Jun 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+`scikit-COSMO` (`skcosmo`) focuses on machine learning methods that are of particular use (but not limited to) the field of materials science and computational chemistry. Currently, many
+machine learning studies employ the foundational `scikit-learn` repository, a
+collection of widely applicable machine learning algorithms and methods.
+Written in the same language (Python), style, and adhering to the same guidelines, we aim to provide
+users the ability to seamlessly include methods tailored for material science in machine learning
+workflows alongside those from `scikit-learn`.
+
+# Statement of need
+
+The `skcosmo` package provides machine learning algorithms developed to meet the needs of a specific community: those who use machine learning to understand or represent collections of atoms or molecules. The importance of particular subclasses of machine learning algorithms varies wildly between different fields; when applying machine
+learning to collections of atoms, there is a comparatively increased emphasis on the representation of data in a numerical format for machine learning pipelines, known as _featurization_. This stems from both the variety of featurization techniques and a the expanse of
+tunable hyperparameters within each technique, and it becomes necessary to compare featurizations, remove redundant features, or condense them. Currently, `skcosmo` contains algorithms developed by the Laboratory of Computational Science and Modeling (COSMO) to address these specific problems, the open-source implementation of which were all previously-unavailable.
```

Comment:
I find this section a little bit clunky. I would propose something like:
```suggestion
While machine learning algorithms are fully general and can be applied to a wide variety of fields, the relative importance of different aspects of machine learning workflows can vary wildly between disciplines. The `skcosmo` package thus provides machine learning algorithms developed to meet the specific needs of those who use machine learning to understand or represent collections of atoms or molecules, where there is increased emphasis on the representation of physical entities---namely, atoms, their connectivity, or their relative orientation in space---in a numerical format compatible with machine learning pipelines, known as the _featurization_. This emphasis stems from both the variety of available featurization techniques and from the expanse of tunable hyperparameters within each technique, necessitating methods for comparing featurizations, removing redundant features, or condensing features. Currently, `skcosmo` contains algorithms developed by the Laboratory of Computational Science and Modeling (COSMO) to address these specific problems, the open-source implementation of which were all previously unavailable.
```

On a somewhat related note, is it "Laboratory of Computational Science and Mode**l**ing" or "Laboratory of Computational Science and Mode**ll**ing"?

Reply:
I like the @bhelfrecht  variant